.github/build.yaml.gomplate

```diff
@@ -575,7 +572,7 @@
   {{{ $config := (datasource "config") }}}
   {{{ $flavor := index . "flavor" }}}
   {{{ $subset := index . "subset" }}}
-  {{{- range slice "efi" "bios" }}}
+  {{{- range slice "bios" }}}
```

Comment:
isnt this the other way around??

Reply:
this what I wanted to ask you, did we already shitf everything to EFI?

Reply:
I think so, bios deprecated, efi by default, we leave the option to have bios in there but not support it no?

Reply:
yes, but I mean, are our VMs in the CI for all tests booting in EFI?

Reply:
good point... pretty sure packer isnt even ready, at least the x86 part. aarch64 is efi as that is the only possible way but not the x86.

And the same for the tests, vagrant for aarch64 is efi ready as there is no bios on aaarch64, but x86 it just uses virtuabox with defaults, so no, its bios.

The change its "easy" but tweaking is gonna be a nightmare. In any case, I would go full EFI, even if it breaks the pipeline. Then we can rework the pipeline to work with that, drop uneeded stuff (bye bye vbox packer) etc..

Reply:
oh you are brave man :wink: 

You convinced me, lets assume a broken pipe for now and move to EFI on follow up PRs until it is fixed.

Reply:
Already testing going the QEMU only path for everything with EFI: https://github.com/rancher/elemental-toolkit/pull/1624 ;)